src/pkg/message/interactive.go

```diff
@@ -0,0 +1,20 @@
+package message
+
+import (
+	"fmt"
+
+	"github.com/pterm/pterm"
+)
+
+func PromptForConfirmation(spinner *Spinner) bool {
+	// Prompt the user to confirm the action
+	if spinner != nil {
+		spinnerText := spinner.Pause()
+		defer spinner.Updatef(fmt.Sprintf("%s\n", spinnerText))
+	}
+
+	confirmation := pterm.DefaultInteractiveConfirm.WithDefaultText("Do you want to continue?")
```

Comment:
```suggestion
	confirmation := pterm.DefaultInteractiveConfirm.WithDefaultText("Do you want to run executable validations?")
```

OR similar -> current process makes me think that inputting "no" would cancel all runtime